docs/sources/variables/syntax.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "Variables syntax and types"
+title = "Variables syntax"
```

Comment:
Should this be `Variable syntax`, `Variable’s syntax`, or `Variables’ syntax`? That plural in there sounds weird without genitive.

Reply:
You are right, should be Variable syntax. Missed that the first time I rewrote this.